pkg/api/types.go

```diff
@@ -1736,6 +1762,9 @@ type Container struct {
 	Resources ResourceRequirements
 	// +optional
 	VolumeMounts []VolumeMount
+	// Block devices to be used by the container..
+	// +optional
+	VolumeDevices []*VolumeDevice
```

Comment:
IIUC, every time we add a new field to an existing struct, then we need to make that new field a pointer.  So this should be `*[]VolumeDevice`.

BTW, if this is the case, then we should update the devel [docs](https://github.com/kubernetes/community/blob/master/contributors/devel/api_changes.md#on-compatibility), because they still say that you can add new fields without pointers.